locations/kollage-kollectiv.yml

```diff
@@ -0,0 +1,55 @@
+---
+location: kollage-kollectiv
+location_nice: "Lilli-Henoch-Straße 21, 10405 Berlin"
+latitude: 52.54194
+longitude: 13.43365
+altitude: 10
+contacts:
+  - "https://matrix.to/#/!mLyZEcpSVQkkxDMsIl:matrix.org?via=matrix.org"
+
+hosts:
+  - hostname: kollage-kollectiv-core
+    role: corerouter
+    model: "genexis_pulse-ex400"
+    wireless_profile: freifunk_default
+
+snmp_devices:
+  - hostname: kollage-kollectiv-switch
+    address: 10.248.89.226
+    snmp_profile: edgeswitch
+
+
+ipv6_prefix: "2001:bf7:750:b900::/56"
+
+# reservierte IPs
+# 10.248.90.0/24 DHCP
+# 10.248.89.224/28 MGMT
+# 10.248.88.240/28 Mesh
+
+networks:
+  - vid: 11
+    role: mesh
+    name: kollage_segen
+    prefix: 10.248.88.240/32
+    ipv6_subprefix: -11
+    mesh_metric: 128
```

Comment:
This shouldnt be needed. As we only have one link we dont need to prioritize any link. Additionally we have defaults for certain types of links in place